packages/fast-development-site-react/src/components/toc/toc-item.tsx

```diff
@@ -66,7 +75,10 @@ const style: ComponentStyles<ITocItemManagedClasses, IDevSiteDesignSystem> = {
             position: "absolute",
             left: "0",
             top: `calc((100% / 2) - ${toPx(tocItemActivePipeHeight / 2)})`
-        }
+        },
+        "&:hover": {
+            backgroundColor: "rgba(0, 0, 0, 0.1)"
```

Comment:
Is there a way that we can use the design system here? When we setup theming of this, it will not change while other values will.